ee-9/pom.xml

```diff
@@ -416,6 +417,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>jakarta.security.auth.message</groupId>
+                <artifactId>jakarta.security.auth.message-api</artifactId>
+                <version>${version.jakarta.security.auth.message-api}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
This dependency should be `jakarta.authentication:jakarta.authentication-api:2.0.0` instead, which is already available in the pom, so I would not expect to introduce this variant (2.0.0-RC1) here.

Reply:
fixed; I found out that this can be removed